source/utilities.tex

```diff
@@ -162,23 +162,33 @@
   template<class T2, class T1>
     constexpr const T2&& get(const pair<T1, T2>&& p) noexcept;
 
-  // \ref{pair.piecewise}, pair piecewise construction
+  // \ref{pair.piecewise}, pair piecewise construction%
+\indexlibraryglobal{piecewise_construct_t}%
+\indexlibraryglobal{piecewise_construct}
```

Comment:
These are already indexed, pointing to [pair.piecewise]. Do we usually index them in the header synopsis too?

Reply:
Good point, fixing now.